### DISCUSSION TITLE
Rollout of v38 in Mend hosted app

### DISCUSSION THREAD

**Discussion by rarkins** [renovatebot/renovate#30417] [Request Help] [answered]
We will begin rolling out v38 in the Mend-hosted Renovate app on Sunday 28th July, starting with "opt-in" orgs first including all those in this project.

If you'd like your account or org added to this list of early adopters, please comment below here.

**Comment by rarkins** [comment]
Accounts/orgs to start with:
- renovatebot
- containerbase
- rarkins
- viceice
- visualon

**Reply by TWiStErRob** [reply]
@TWiStErRob

**Comment by TWiStErRob** [comment]
Did this happen? I saw a strange failure a few days ago, but checking the logs today it's still showing 37.x now.

**Reply by viceice** [reply]
@nabeelsaabna did you forgot this?

**Reply by nabeelsaabna** [reply]
V38 released a few minutes ago to renovate-ce
The hosted app update is planned for Sunday (August 4)

**Reply by rarkins** [reply]
@nabeelsaabna Did you add @TWiStErRob to the opt-in testers earlier?

**Reply by nabeelsaabna** [reply]
No, I only saw this issue today

**Reply by TWiStErRob** [reply]
np, thanks for the update

**Reply by nabeelsaabna** [reply]
@TWiStErRob The hosted app is running v38

**Comment by nabeelsaabna** [comment, marked as answer]
Renovate version in the Mend App updated to `v38.18.17`

**Reply by thor** [reply]
Where'd you like possible bug reports; investigation here or straight into the issues?

**Reply by rarkins** [reply]
Maybe best to create a separate discussion for each

**Comment by allanlewis** [comment]
Is there a way to be notified of announcements like this? (Email subscription? Slack?)

**Reply by HonkingGoose** [reply]
We don't have a simple way to get notified for new major releases. I'll let the maintainers decide how/if they want to fix that.

For now, you can subscribe to this issue:

- https://github.com/renovatebot/renovate/issues/25608

This does mean that you must:

- Search for the new major planning issue.
- Subscribe to the new major planning issue.

Subscribing to Renovate's releases on GitHub does work, but notifies you for all releases! It would be nice if you could tell GitHub to only alert you for `major` updates...
